app/views/layouts/_header_user_menu.haml

```diff
@@ -16,6 +16,10 @@
     = icon_map_tag(icons, "list", ["icon-with-text"])
     = t("header.manage_listings")
 
+  = link_to working_date_slots_path do
+    = icon_map_tag(icons, "calendar", ["icon-with-text"])
+    = t("header.manage_availability")
```

Comment:
Perhaps we should use `translate` instead of `t`? At the very least it would be easier to find the translations we've created.

Reply:
I went a bit back and forth on this, and while I'd prefer to use the full method name, I figured going for consistency with the other elements in the file was the better option.